lib/token-pool.js

```diff
@@ -219,6 +219,12 @@ class TokenPool {
     this.fifoQueue.forEach(visit);
     this.priorityQueue.forEach(visit);
   }
+
+  allValidTokenIds() {
```

Comment:
Simple code, but adding a small test to our existing suite wouldn't hurt! 😄

Reply:
Done!